index.js

```diff
@@ -90,6 +113,11 @@ module.exports = {
             context.revisions = revisions;
           });
       },
+
+      _errorMessage: function(error) {
```

Comment:
We can safely remove that I guess as it isn't used anymore in the latest changes.